packages/tame-metering/src/tame.js

```diff
@@ -71,12 +74,19 @@ export function tameMetering() {
 
     let targetIsConstructor = false;
 
-    // Without this hack, SES-1.0 fails with:
+    // Without this ErrorConstructor hack, SES pre-0.8 fails with:
     // TypeError: prototype function Error() { [native code] } of unknown.global.EvalError is not already in the fringeSet
+    // This is due to an ordering constraint between when SES-pre-0.8 captures the intrinsics versus when it runs the shims.
+    //
+    // SES 0.8 and later do not have this constraint, since the vetted shims are installed before SES is even imported.
+    //
+    // Not wrapping the Error constructor doesn't cause much of an exposure, although it can be used to allocate large
+    // strings without counting towards the allocation meter.  That's not a problem we're trying to solve yet.
```

Comment:
And if the target is `globalEval`, I presume this doesn’t violate containment because that eval is in turn a wrapper that will rewrite the given program to incorporate the meter. Is that correct?

Reply:
Yes, it is only used internally by SES, and otherwise is replaced with the wrapper.